code/modules/mob/living/carbon/human/species_types/ethereal.dm

```diff
@@ -96,7 +96,7 @@
 		var/healthpercent = max(H.health, 0) / 100
 		if(!emageffect)
 			current_color = rgb(r2 + ((r1-r2)*healthpercent), g2 + ((g1-g2)*healthpercent), b2 + ((b1-b2)*healthpercent))
-		ethereal_light.set_light_range_power_color(1 + (2 * healthpercent), 1 + (1 * healthpercent), current_color)
+		ethereal_light.set_light_range_power_color(2 + (3 * healthpercent), 2 + (2 * healthpercent), current_color) //NSV13- Its not a performance issue here, and we're theoretically balanced for PVE.
```

Comment:
Remove the commentary.

Reply:
The commentary is so that when some inevitable bee-base change comes along, whomever does the change can look and see why it was increased. 

Namely that we do not have (and to my knowledge, never have had) sufficient ethereal players to where this is a performance issue, and we're balanced for PVE so its not a balance issue.

I do not understand the need to remove a comment providing said detail. 

Does it make the code run less efficiently? Should we remove other comments that provide details? 

I found a literal poem commented out while I was hunting through the wrong place to find the variables I was looking to modify. Should I find that again and delete it?

I will change it if there is good reason to do so. If you still don't like the fact that I added said detail but do not have a logical, code-based reason to do so, close the PR.

Reply:
Yes, and the tl;dr above belongs in the PR "Why It's Good For The Game" section which gets hooked and referenced through plugins like GitLense so we can all see exactly which PR made changes and why.

However if you want to make it a fight, please do battle with the close PR button yourself?

Reply:
Neither of these is a good reason for closing a PR so let's all quit daring each other to do it, okay?

What would be more helpful to me when beebasing if there's going to be a comment is describing the function of the change (e.g. "made ethereals brighter") in case the block of code gets moved, but we've got plenty of uncommented tags lying around too.

Reply:
Comment changed to desired descriptive details. Let me know if you want it to be more succinct, corvid.

Reply:
Can't see it yet, make sure you push